MediaSystem.cpp

```diff
@@ -32,34 +32,6 @@ using namespace WPEFramework;
 
 namespace CDMi {
 static constexpr const TCHAR ControllerCallsign[] = _T("Controller");
```

Comment:
is this static being used other than within the ControllerLink?

Reply:
yes, it is redundant, so removed